__tests__/components/editor/CopyToNewMessage.test.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import React from 'react'
+import { createStore } from 'redux'
+import CopyToNewMessage from 'components/editor/CopyToNewMessage'
+import appReducer from 'reducers/index'
+/* eslint import/no-unresolved: 'off' */
+import { renderWithRedux } from 'testUtils'
+import { getNodeText } from '@testing-library/react'
+
+test('Message is not displayed when show is false', () => {
+  const store = createStore(appReducer, {
+    selectorReducer: {
+      entities: {},
+      resource: {},
+      editor: {
+        copyToNewMessage: {
+          show: false,
+        },
+      },
+    },
+  })
+  const { queryByText } = renderWithRedux(
+    <CopyToNewMessage />, store,
+  )
+  expect(queryByText('Copied to new resource')).not.toBeInTheDocument()
+})
+
+test('Default message is displayed when show in state is true', () => {
+  const store = createStore(appReducer, {
+    selectorReducer: {
+      entities: {},
+      resource: {},
+      editor: {
+        copyToNewMessage: {
+          show: true,
+        },
+      },
+    },
+  })
+  const { container } = renderWithRedux(
+    <CopyToNewMessage />, store,
+  )
+  expect(container.querySelector('.alert-info')).toBeInTheDocument()
```

Comment:
Would be preferable to search for text rather than class.

Reply:
I'm confused. What is being displayed here?

Reply:
This just displays a short message of what the source resource URI used for the copy.

Reply:
Removed this particular test as it no longer being displayed if a user hits copy on an unsaved resource.